_posts/2017-02-14-planning-for-7.0.md

```diff
@@ -0,0 +1,194 @@
+
+You might remember that after Babel 6, Babel became a set of npm packages with its own ecosystem of custom presets and plugins.
+
+However since then, we've always used a "fixed/synchronized" versioning system (so that no package is on v7.0 or above). When we do a new release such as `v6.23.0` only packages that have updated code in the source are published with the new version while the rest of the packages are left as is. This mostly works in practice because we use `^` in our packages.
+
+Unfortunately this kind of system requires that a major version be released for all packages once a single package needs it. This either means we make a lot small breaking changes (unnecessary) or we batch lots of breaking changes into a single release. This is the same issue for a single package/repo but we have a way out: since Babel is a monorepo of npm packages, we can actually publish breaking changes in individual packages if necessary.
+
+This just means that we intend to make major version bumps to any experimental proposal plugins when the spec changes rather than waiting to update all of Babel. So anything that is < Stage 4 would be open to breaking changes in the form of major version bumps and same with the Stage presets themselves if we don't drop them entirely. 
+
+## 🚀 New Feature
+
+### [#3683](https://github.com/babel/babel/pull/3683) Add `transform-unicode-property-regex` to the Stage 2 preset
+
+[transform-unicode-property-regex](https://github.com/mathiasbynens/babel-plugin-transform-unicode-property-regex) is maintained by [@mathiasbynens](https://github.com/mathiasbynens)
+
+> Compile Unicode property escapes (\p{…} and \P{…}) in Unicode regular expressions to ES5 or ES6 that works in today’s environments.
+
+Input
+
+```js
+var regex = /\p{ASCII_Hex_Digit}/u;
+```
+
+Output
+
+```js
+var regex = /[0-9A-Fa-f]/;
+```
+
+## 🤔 Questions
+
+### Using npm Scoped Packages
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Thoughts on <a href="https://twitter.com/babeljs">@babeljs</a> using npm scoped packages for 7.0?</p>&mdash; Henry Zhu (@left_pad) <a href="https://twitter.com/left_pad/status/821551189166878722">January 18, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+Seems like most who understood what scoped packages were was in favor?
+
+Pros
+
+- Don't have to worry about getting a certain package name (the reason why this was brought up in the first place).
+
+> Many package names have been taken (preset-es2016, preset-es2017, 2020, 2040, etc). Can always ask to transfer but not always easy to do and can lead to users believing certain packages are official due to the naming.
+
+Cons
+
+- We need to migrate to new syntax
+- Still unsupported on certain non-npm tools (lock-in)
+- No download counts unless we alias back to old names
+
+Sounds like we may want to defer, and in the very least it's not a breaking change given it's a name change.
```

Comment:
Also you could raise these concerns with npm as a potential blocker for adoption.